examples/linear-regression/Dockerfile

```diff
@@ -0,0 +1,12 @@
+
+FROM golang:alpine
+RUN  apk update && apk add make linux-headers git
+RUN mkdir go/src/github.com/Soypete -p
+RUN cd go/src/github.com/Soypete
+CMD ls
+RUN git clone https://github.com/Soypete/golearn.git
+
+RUN cd golearn/examples/linear-regression
+CMD ls
+CMD go build ./...
+ENTRYPOINT [ ./linear-regression ]
```

Comment:
should this need to be `CMD`?

Reply:
so I am actually this[ file to](https://github.com/Soypete/golearn/pull/3/files#diff-3254677a7917c6c01f55212f86c57fbf) build the model ... I can delete the other file I guess